systems/WikibaseClient/03-Context_and_Scope.md

```diff
@@ -12,3 +12,12 @@
 ### Technical Context
 
 ![WikibaseClient technical context diagram](./diagrams/03-technical-context.drawio.svg)
+
+| Component                                                                     | Description                                                                                                                                                              |
+| ----------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Lua/Parser function                                                           | [Lua bindings](../overview/12-Glossary.md#lua) or parser functions invoked through [Wikitext](../overview/12-Glossary.md#wikitext) to embed Entity data in Wiki articles |
+| [ChangeHandler](./05-Building_Block_View.md#entity-change-notifications)      | Receives notifications to update pages when an Item is changed on the Repo                                                                                               |
```

Comment:
the diagram says "and purges" - is that worth mentioning here?

Reply:
In a way purging is part of the update process, because it wouldn't become visible otherwise, but it makes sense to mention it. Added!